From 614a4a2f74e8a6d14a43853c2ce134358b5e86f3 Mon Sep 17 00:00:00 2001
From: Valentine Barshak <valentine.barshak@cogentembedded.com>
Date: Tue, 25 Jun 2019 23:45:40 +0300
Subject: [PATCH 122/211] arm64: dts: renesas: r8a77980: Convert VIN nodes to
 soc_camera model

This converts VIN nodes to soc_camera model.

Signed-off-by: Valentine Barshak <valentine.barshak@cogentembedded.com>
---
 arch/arm64/boot/dts/renesas/r8a77980.dtsi | 136 ------------------------------
 1 file changed, 136 deletions(-)

diff --git a/arch/arm64/boot/dts/renesas/r8a77980.dtsi b/arch/arm64/boot/dts/renesas/r8a77980.dtsi
index 1b56b1f..63c49a8 100644
--- a/arch/arm64/boot/dts/renesas/r8a77980.dtsi
+++ b/arch/arm64/boot/dts/renesas/r8a77980.dtsi
@@ -867,23 +867,6 @@
 			resets = <&cpg 811>;
 			renesas,id = <0>;
 			status = "disabled";
-
-			ports {
-				#address-cells = <1>;
-				#size-cells = <0>;
-
-				port@1 {
-					#address-cells = <1>;
-					#size-cells = <0>;
-
-					reg = <1>;
-
-					vin0csi40: endpoint@2 {
-						reg = <2>;
-						remote-endpoint= <&csi40vin0>;
-					};
-				};
-			};
 		};
 
 		vin1: video@e6ef1000 {
@@ -895,23 +878,6 @@
 			status = "disabled";
 			renesas,id = <1>;
 			resets = <&cpg 810>;
-
-			ports {
-				#address-cells = <1>;
-				#size-cells = <0>;
-
-				port@1 {
-					#address-cells = <1>;
-					#size-cells = <0>;
-
-					reg = <1>;
-
-					vin1csi40: endpoint@2 {
-						reg = <2>;
-						remote-endpoint= <&csi40vin1>;
-					};
-				};
-			};
 		};
 
 		vin2: video@e6ef2000 {
@@ -923,23 +889,6 @@
 			resets = <&cpg 809>;
 			renesas,id = <2>;
 			status = "disabled";
-
-			ports {
-				#address-cells = <1>;
-				#size-cells = <0>;
-
-				port@1 {
-					#address-cells = <1>;
-					#size-cells = <0>;
-
-					reg = <1>;
-
-					vin2csi40: endpoint@2 {
-						reg = <2>;
-						remote-endpoint= <&csi40vin2>;
-					};
-				};
-			};
 		};
 
 		vin3: video@e6ef3000 {
@@ -951,23 +900,6 @@
 			resets = <&cpg 808>;
 			renesas,id = <3>;
 			status = "disabled";
-
-			ports {
-				#address-cells = <1>;
-				#size-cells = <0>;
-
-				port@1 {
-					#address-cells = <1>;
-					#size-cells = <0>;
-
-					reg = <1>;
-
-					vin3csi40: endpoint@2 {
-						reg = <2>;
-						remote-endpoint= <&csi40vin3>;
-					};
-				};
-			};
 		};
 
 		vin4: video@e6ef4000 {
@@ -979,23 +911,6 @@
 			resets = <&cpg 807>;
 			renesas,id = <4>;
 			status = "disabled";
-
-			ports {
-				#address-cells = <1>;
-				#size-cells = <0>;
-
-				port@1 {
-					#address-cells = <1>;
-					#size-cells = <0>;
-
-					reg = <1>;
-
-					vin4csi41: endpoint@2 {
-						reg = <2>;
-						remote-endpoint= <&csi41vin4>;
-					};
-				};
-			};
 		};
 
 		vin5: video@e6ef5000 {
@@ -1007,23 +922,6 @@
 			resets = <&cpg 806>;
 			renesas,id = <5>;
 			status = "disabled";
-
-			ports {
-				#address-cells = <1>;
-				#size-cells = <0>;
-
-				port@1 {
-					#address-cells = <1>;
-					#size-cells = <0>;
-
-					reg = <1>;
-
-					vin5csi41: endpoint@2 {
-						reg = <2>;
-						remote-endpoint= <&csi41vin5>;
-					};
-				};
-			};
 		};
 
 		vin6: video@e6ef6000 {
@@ -1035,23 +933,6 @@
 			resets = <&cpg 805>;
 			renesas,id = <6>;
 			status = "disabled";
-
-			ports {
-				#address-cells = <1>;
-				#size-cells = <0>;
-
-				port@1 {
-					#address-cells = <1>;
-					#size-cells = <0>;
-
-					reg = <1>;
-
-					vin6csi41: endpoint@2 {
-						reg = <2>;
-						remote-endpoint= <&csi41vin6>;
-					};
-				};
-			};
 		};
 
 		vin7: video@e6ef7000 {
@@ -1063,23 +944,6 @@
 			resets = <&cpg 804>;
 			renesas,id = <7>;
 			status = "disabled";
-
-			ports {
-				#address-cells = <1>;
-				#size-cells = <0>;
-
-				port@1 {
-					#address-cells = <1>;
-					#size-cells = <0>;
-
-					reg = <1>;
-
-					vin7csi41: endpoint@2 {
-						reg = <2>;
-						remote-endpoint= <&csi41vin7>;
-					};
-				};
-			};
 		};
 
 		vin8: video@e6ef8000 {
-- 
2.7.4

